_tutorials/2016-05-22-getting-started-on-xrv64.md

```diff
@@ -0,0 +1,118 @@
+---
+published: false
+date: "2016-05-22 13:49 -0700"
+title: Getting Started on XRv64
+permalink: "https://xrdocs.github.io/application-hosting/tutorials/xrv64-quickstart"
+author: Lisa Roach
+excerpt: "Getting started with Cisco's XRv64 Vagrant Instance."
+tags: 
+  - vagrant
+  - iosxr
+  - cisco
+  - XRv64
+---
+### Quick Start Using XRv64 Box image
+Quick guide to get up and running with XRv64 and vagrant/Virtualbox.
+ 
+Install Virtualbox and vagrant tools to your mac:
+
+Source: http://sourabhbajaj.com/mac-setup/Vagrant/README.html
+
+*	Recommend using the Homebrew package manager.
+
+*	Make sure you install version 5.x virtualbox
+
+*	Vagrant latest version is: 1.8.2
+
+*	Website:  http://brew.sh/
+
+        /usr/bin/ruby -e "$(curl -fsSLhttps://raw.githubusercontent.com/Homebrew/install/master/install)"
+        brew cask install virtualbox
+        brew cask install vagrant
+        brew install socat
+
+Note you may need to install pexpect:
+
+	brew cask install python
+	pip install pexpect
+
+#### Single Node:
+
+	vagrant init xrv64
+  ![vagrant init xrv64](https://xrdocs.github.io/xrdocs-images/assets/images/xrv64_vagrant_init.png)
```

Comment:
I believe you've uploaded all the images to the tutorial-images folder? There is a direct correlation with the asset folder you upload to, so

```
 If you uploaded to a specific directory like 
 https://github.com/xrdocs/xrdocs-images/upload/gh-pages/assets/tutorial-images
 Then the image may be refrenced as:
 ![<image context>](https://xrdocs.github.io/xrdocs-images/assets/tutorial-images/<image-name.jpg/png/jpeg>)
 ```